charmhelpers/contrib/openstack/ip.py

```diff
@@ -139,9 +142,21 @@ def local_address(unit_get_fallback='public-address'):
     :returns: IP Address
     :rtype: str
     """
+    log(
+        "local_address is deprecated. Please find the call and replace it with "
+        "a network-get call with an explicit binding.", "WARNING")
```

Comment:
If you wanted to be REALLY interesting/nasty here, you could use `trace_back` and print the stack at this point to tell them where the call is :)  It might make the logs a bit messy, but would highlight the issue.